### PR TITLE
TropiCata: Birds Galore

### PR DESCRIPTION
**Pull request by Xaritscin**
#### Summary
Mods "Adds several species of tropical and subptropical birds"

#### Purpose of change

The list of birds added on mod was a bit old and needed some expanding. This update brings a whole lot more of birds to populate the forests, plains and waters of the tropical region.

#### Describe the solution

-added hummingbird
-added woodpecker
-added toucan
-added parrot
-added budgerigar (also known as parakeet)
-added cockatoo
-added macaw
-added tinamou
-added seriema
-added chachalaca
-added guan
-added curassow
-added flamingo
-added coot
-added cormorant
-added moorhen
-added grebe

all birds so far have an egg and chick form. im not an ornitologist so i tried to do my best and guess their volume and use average weights from the internet to fill up their stats.

tinamous, chachalacas, guans and curassows replace the grouse and pheasants of the vanilla game and are also alternative game to turkeys. Flamingos along with coots, moorhens, grebes and cormorants are additions to aquatic biomes like rivers, lakes and marshes so as to cut down on the duck/geese spam along with reducing the chance roll of other more dangerous fauna. Parrots, budgies, cockatoos and macaws should be able to be tamed and do tricks for the player. I have no plans for now related to toucans, seriemas, woodpeckers and hummingbirds but they are a good additional sight in their respective biomes.

#### Describe alternatives you've considered

the current list of birds may be sufficient after summing it up with the non blacklisted vanilla stock.

#### Testing

generated a new map for testing. found the birds around their respective biomes. spawned a few parrots and budgies to feed them with bird food. they became friendly and could play with them.

#### Additional context

was planning to add some more birds like owls, potoos, hoatzins and/or quetzals but i think it was overstretching.

### CODE REVIEW

File: data/mods/TropiCataclysm/monsters/tropical_bird.json [hunk 1]
```diff
@@ -174,5 +174,271 @@
...
+    "reproduction": { "baby_egg": "egg_hummingbird", "baby_count": 5, "baby_timer": 14 },
+    "baby_flags": [ "SPRING" ],
+    "biosignature": { "biosig_item": "feces_bird", "biosig_timer": 8 },
+    "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "FLIES" ]
+  },
+  {
+    "id": "mon_woodpecker",
+    "type": "MONSTER",
+    "name": { "str": "woodpecker" },
+    "description": "A small bird with woody colors and a colored crest on its head.  These birds are well known for their hability to poke holes on trees while looking for grubs.",
```
Comment (casswedson) [inline data/mods/TropiCataclysm/monsters/tropical_bird.json:211]:
```suggestion
    "description": "A small bird with woody colors and a colored crest on its head.  These birds are well known for their ability to poke holes on trees while looking for grubs.",
```

File: data/mods/TropiCataclysm/monsters/tropical_bird.json [hunk 2]
```diff
@@ -174,5 +174,271 @@
...
+    "reproduction": { "baby_egg": "egg_woodpecker", "baby_count": 5, "baby_timer": 14 },
+    "baby_flags": [ "SPRING" ],
+    "biosignature": { "biosig_item": "feces_bird", "biosig_timer": 8 },
+    "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "FLIES" ]
+  },
+  {
+    "id": "mon_toucan",
+    "type": "MONSTER",
+    "name": { "str": "toucan" },
+    "description": "A small bird with with dark plumage and a bright colored face and beak.  These tropical birds can be found all around the Americas feeding on fruits and seeds.",
```
Comment (casswedson) [inline data/mods/TropiCataclysm/monsters/tropical_bird.json:240]:
```suggestion
    "description": "A small bird with dark plumage and a bright colored face and beak.  These tropical birds can be found all around the Americas feeding on fruits and seeds.",
```

File: data/mods/TropiCataclysm/monsters/tropical_bird.json [hunk 3]
```diff
@@ -174,5 +174,271 @@
...
+    "reproduction": { "baby_egg": "egg_cockatoo", "baby_count": 5, "baby_timer": 14 },
+    "baby_flags": [ "SPRING" ],
+    "biosignature": { "biosig_item": "feces_bird", "biosig_timer": 8 },
+    "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "FLIES", "SWARMS" ]
+  },
+  {
+    "id": "mon_budgerigar",
+    "type": "MONSTER",
+    "name": { "str": "budgerigar" },
+    "description": "A small bird with with bright colored plumage and a short hooked beak.  These small relatives of the parrot come from Australia but were domesticated worldwide as pets.",
```
Comment (casswedson) [inline data/mods/TropiCataclysm/monsters/tropical_bird.json:327]:
```suggestion
    "description": "A small bird with bright colored plumage and a short hooked beak.  These small relatives of the parrot come from Australia but were domesticated worldwide as pets.",
```

File: data/mods/TropiCataclysm/monsters/tropical_bird.json [hunk 4]
```diff
@@ -174,5 +174,271 @@
...
+    "reproduction": { "baby_egg": "egg_budgerigar", "baby_count": 5, "baby_timer": 14 },
+    "baby_flags": [ "SPRING" ],
+    "biosignature": { "biosig_item": "feces_bird", "biosig_timer": 8 },
+    "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "FLIES", "SWARMS" ]
+  },
+  {
+    "id": "mon_macaw",
+    "type": "MONSTER",
+    "name": { "str": "macaw" },
+    "description": "A medium to large bird with with bright colored plumage and a short hooked beak.  These are the largest flying parrots in the world, living usually in the thickest rainforests.",
```
Comment (casswedson) [inline data/mods/TropiCataclysm/monsters/tropical_bird.json:356]:
```suggestion
    "description": "A medium to large bird with bright colored plumage and a short hooked beak.  These are the largest flying parrots in the world, living usually in the thickest rainforests.",
```

**Comment by Xaritscin** [comment]
ready for review i think

**Pull request merged**